src/Analysis/AnalysisApp.jsx

```diff
@@ -9,6 +9,9 @@ import HIVCohortFilter from '../HIVCohortFilter/HIVCohortFilter';
 import ReduxGWASUIApp from './GWASUIApp/ReduxGWASUIApp';
 import { analysisApps } from '../localconf';
 import './AnalysisApp.css';
+import sessionMonitor from '../SessionMonitor';
+// monitor user's session
+sessionMonitor.start();
```

Comment:
why do we need this?

Reply:
I think you are right. I disabled it and it still seems to work.